packages/kit/src/runtime/client/router.js

```diff
@@ -72,6 +73,29 @@ export class Router {
 		};
 	}
 
+	save_scroll_position() {
+		sessionStorage.setItem(
+			`sveltekit:scroll:${this.current_history_index}`,
+			JSON.stringify(scroll_state())
+		);
+	}
+
+	/**
+	 * @param {number} index
+	 * @returns {{x: number, y: number} | null}
+	 */
+	get_scroll_position(index) {
```

Comment:
can we either make this method and `save_scroll_position` a standalone function next to `scroll_state` or prefix with `#` to make them private? I don't think they need to be exposed outside of this class

Reply:
done